tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -0,0 +1,86 @@
+*** Settings ***
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHAdminUI.robot
+Resource        ../../Resources/ODS.robot
+Suite Setup     Set Library Search Order  SeleniumLibrary
+Suite Teardown  Set Default Access Groups Settings
+
+*** Test Cases ***
+Verify if the CRD after applying changed In admin UI
+    [Documentation]  Verify if the CRD after applying changed In admin UI
+    [Tags]  ODS-1661
+    ...     Tier1
+    ...     Sanity
+    Launch Dashboard And Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear User Management Settings
+    Add OpenShift Groups To Data Science Administrators   rhods-admins  rhods-users
+    Add OpenShift Groups To Data Science User Groups    system:authenticated
+    Save Changes In User Management Setting
+    AdminGroups In OdhDashboardConfig CRD Should Be     rhods-admins  rhods-users
+    AllowedGroups In OdhDashboardConfig CRD Should Be   system:authenticated
+
+Verify If Unauthorisez User Can Not Change The Permission
+    [Documentation]  Verify If Unauthorisez User Can Not Change The Permission
+    ...  if Unauthorisez user has saved the changes should not refect In CRD
+    [Tags]  ODS-1660
+    ...     ODS-1555
+    ...     Tier1
+    ...     Sanity
+    Launch Dashboard And Check User Management Option Is Available For The User     ${TEST_USER_3.USERNAME}   ${TEST_USER_3.PASSWORD}  ${TEST_USER_3.AUTH_TYPE}
+    Remove OpenShift Groups From Data Science Administrator Groups     rhods-admins
+    Save Changes In User Management Setting
+    Switch Browser  1
+    Add OpenShift Groups To Data Science Administrators    rhods-noaccess
+    Add OpenShift Groups To Data Science User Groups       rhods-noaccess
+
+    AdminGroups In OdhDashboardConfig CRD Should Be        rhods-users
+    AllowedGroups In OdhDashboardConfig CRD Should Be      system:authenticated
+    Save Changes In User Management Setting
+    Page Should Contain  Unable to load User and group settings
+    Switch Browser  2
+    [Teardown]  Teardown For Admin UI   ${TEST_USER_3.USERNAME}   ${TEST_USER_3.PASSWORD}  ${TEST_USER_3.AUTH_TYPE}
+
+Verify Unauthorisez User Is Not Able To Spawn Jupyter Notebook
+    [Documentation]    Verify User Is Not Able To Spawn Jupyter Notebook
```

Comment:
could you give more information in the documentation? Like when the user shouldn't be able to spawn the server

Reply:
could u check now?

Reply:
better! thanks :)